src/translations/en.json

```diff
@@ -265,6 +265,8 @@
   "vlId": "Voltage level ID",
   "p0": "Constant P",
   "q0": "Constant Q",
+  "batteryTargetP": "Constant active power",
```

Comment:
US are updated recently by Chistine, we do not need these keys, can reuse targetP, targetQ

In addition, we need another key, ie. 'filterNominalVoltage: Voltage level nominal voltage' which is different to the existing key nominalVoltage

Reply:
Done.